### ISSUE TITLE
dblab-sqlite3 does not work on OSX

### ISSUE THREAD

**Issue by kascote** [danvergara/dblab#113]
**Describe the bug**
when try to execute `dblab-sqlite` on OSX gives this error
```
Error: Binary was compiled with 'CGO_ENABLED=0', go-sqlite3 requires cgo to work. This is a stub
```

**To Reproduce**
1. Download https://github.com/danvergara/dblab/releases/download/v0.16.0/dblab_0.16.0_darwin_arm64.tar.gz
2. extract and move executable to a directory on path
3. execute it

**Expected behavior**
open the database and show the content

**Screenshots**
<img width="1068" alt="Screen Shot 2022-09-24 at 19 34 33" src="https://user-images.githubusercontent.com/9757/192120949-97e6e781-bb8e-4238-9611-bdc967d2db34.png">

**OS (please complete the following information):**
❱ uname -a
Darwin mary3jane.local 21.6.0 Darwin Kernel Version 21.6.0: Mon Aug 22 20:19:52 PDT 2022; root:xnu-8020.140.49~2/RELEASE_ARM64_T6000 arm64

**Additional context**

**Comment by danvergara** [comment]
A release candidate for a bugfix has been shipped, Please, go check it out!
https://github.com/danvergara/dblab/releases/tag/untagged-663792a1cf9ae3c9e192

**Comment by kascote** [comment]
seems that `rc-1` doesn't include the sqlite version


<img width="575" alt="Screen Shot 2022-09-29 at 20 44 09" src="https://user-images.githubusercontent.com/9757/193160850-9ba4db93-463b-4afc-a6a2-6641634d2779.png">

**Comment by danvergara** [comment]
@kascote thanks, already on it. It has to do with a misconfiguration we have in the goreleaser setup.

**Comment by danvergara** [comment]
https://github.com/danvergara/dblab/releases/tag/v0.16.1-rc1

Try again, please. Note that you can find a different compressed file for every flavor of the binary (regular and sqlite). Not in the same compressed file anymore.

**Comment by kascote** [comment]
now the app starts but works strange. I can't see the table contents/structure. I can write a sql query but doesn't execute, there is something odd with some keybindings, I tried with Kitty and Terminal app (btw on Terminal the app freeze after trying to execute the sql query). I know this isn't a proper but report, but I don't have time now for a correct review.
I think for now this ticket is fixed. On the weekend I'll try to take another shot and may be wrote some ticket. Thanks!

**Comment by danvergara** [comment]
@kascote got it! I'm closing this since we released the [v0.16.1](https://github.com/danvergara/dblab/releases/tag/v0.16.1) patch and it is fixing the bug mentioned above.

Please, free to open a new issue with further details on what's happening on your side.

Thanks for all the support.
